js/privatebin.js

```diff
@@ -2123,149 +2136,162 @@ jQuery.PrivateBin = function($, sjcl, Base64, RawDeflate) {
         me.moveAttachmentTo = function($element, label)
         {
             // move elemement to new place
-            me.$attachmentLink.appendTo($element);
+            $attachmentLink.appendTo($element);
 
             // update text
-            I18n._(me.$attachmentLink, label, me.$attachmentLink.attr('download'));
-        }
-
+            I18n._($attachmentLink, label, $attachmentLink.attr('download'));
+        };
 
-        /*
-         * Read file data as dataURL using the FileReader API
-         * https://developer.mozilla.org/en-US/docs/Web/API/FileReader#readAsDataURL()
+        /**
+         * Read file data as dataURL using the FileReader API.
+         * @param {type} aFile The loaded file.
+         * @see {@link https://developer.mozilla.org/en-US/docs/Web/API/FileReader#readAsDataURL()}
+         * @returns {undefined}
          */
-        me.readFileData = function (file) {
+        me.readFileData = function (aFile) {
             if (typeof FileReader === 'undefined') {
                 // revert loading status…
-                AttachmentViewer.hideAttachment();
-                AttachmentViewer.hideAttachmentPreview();
-                Alert.showError(I18n._('Your browser does not support uploading encrypted files. Please use a newer browser.'));
+                me.hideAttachment();
+                me.hideAttachmentPreview();
+                Alert.showError('Your browser does not support uploading encrypted files. Please use a newer browser.');
                 return;
             }
 
             var fileReader = new FileReader();
-            if (file === undefined) {
-                file = me.$fileInput[0].files[0];
-                $('#dragAndDropFileName').text('');
+            if (aFile === undefined) {
+                aFile = $fileInput[0].files[0];
+                $($dragAndDropFileName).text('');
             } else {
-                $('#dragAndDropFileName').text(file.name);
+                $($dragAndDropFileName).text(aFile.name);
             }
 
-            me.file = file;
+            file = aFile;
 
             fileReader.onload = function (event) {
                 var dataURL = event.target.result;
-                me.attachmentData = dataURL;
+                attachmentData = dataURL;
 
                 if (Editor.isPreview()) {
-                    me.handleAttachmentPreview(AttachmentViewer.$attachmentPreview, dataURL);
+                    me.handleAttachmentPreview($attachmentPreview, dataURL);
                 }
             };
-            fileReader.readAsDataURL(file);
+            fileReader.readAsDataURL(aFile);
         };
 
         /**
          *  Handle the preview of files that can either be an image, video, audio or pdf element.
-         *  @argument {DOM Element} targetElement where the preview should be appended.
+         *  @argument {DOM Element} $targetElement where the preview should be appended.
```

Comment:
Upps, sorry. If the element is really only a DOM element and not a jQuery element, you must of course not prepend it with $. Maybe, I've misunderstood the code here, sorry.
Currently the JSDOC here is a bit contrary: `DOM Element <-> $…`

If I see it correctly better document a jQuery object – if it is one! – as `{jQuery}`.